backend/infrahub/core/diff/model/path.py

```diff
@@ -753,6 +756,14 @@ def branches(self) -> set[str]:
             for database_edge in (self.path_to_node, self.path_to_attribute, self.path_to_property)
         }
 
+    @property
+    def all_branches(self) -> tuple[str, str, str]:
+        return (
+            self.path_to_node.get("branch"),
+            self.path_to_attribute.get("branch"),
+            self.path_to_property.get("branch"),
+        )
```

Comment:
When using this property would the order of these tuples matter in any way? I.e. would the caller have to know how the first `str` differs from the other two?